umpf

```diff
@@ -288,6 +304,18 @@ setup() {
 	if [ -n "${GIT_FALLBACK_REMOTE}" ]; then
 		GIT_FALLBACK_REMOTE="${GIT_FALLBACK_REMOTE}/"
 	fi
+
+	for topic in "${!OVERRIDES[@]}"; do
+		local rev
+
+		rev="$(${GIT} rev-parse "${GIT_REMOTE}${OVERRIDES[$topic]}^{}" 2> /dev/null)"
+		if [ -z "${rev}" ]; then
+			echo "Revision ${GIT_REMOTE}${OVERRIDES[$topic]} not found for topic ${name}"
```

Comment:
And do this here:
```suggestion
		if [ -z "${OVERRIDES[$topic]}" ]; then
			OVERRIDES[$topic]="${GIT_REMOTE}/${topic}"
		fi            
		rev="$(${GIT} rev-parse "${OVERRIDES[$topic]}^{}" 2> /dev/null)"
		if [ -z "${rev}" ]; then
			echo "Revision ${OVERRIDES[$topic]} not found for topic ${topic}"
```

The remote should only be added to the implicit branch, not the explicit overwrite. So I can do this:

```
umpf tag --remote=foo --overwrite=topic/bar:topic/bar-wip
```
So use most branches from remote but one local overwrite from a wip branch.

And this way it is consistent with `umpf merge`. Here the argument is also the full commit-ish including the remote.

Note the change of the last variable in the error message. I'm pretty sure '${name}' is not something valid here.